src/hugr/rewrite/outline_cfg.rs

```diff
@@ -227,19 +239,25 @@ mod test {
     use std::collections::HashSet;
 
     use crate::algorithm::nest_cfgs::test::{
-        build_cond_then_loop_cfg, build_conditional_in_loop_cfg,
+        build_cond_then_loop_cfg, build_conditional_in_loop, build_conditional_in_loop_cfg,
+    };
+    use crate::builder::{
+        Container, Dataflow, DataflowSubContainer, HugrBuilder, ModuleBuilder, SubContainer,
     };
+    use crate::extension::prelude::USIZE_T;
     use crate::extension::PRELUDE_REGISTRY;
+    use crate::hugr::views::sibling::SiblingMut;
     use crate::hugr::HugrMut;
-    use crate::ops::handle::NodeHandle;
-    use crate::{HugrView, Node};
+    use crate::ops::handle::{BasicBlockID, NodeHandle};
+    use crate::types::FunctionType;
+    use crate::{type_row, HugrView, Node};
     use cool_asserts::assert_matches;
     use itertools::Itertools;
 
     use super::{OutlineCfg, OutlineCfgError};
 
     fn depth(h: &impl HugrView, n: Node) -> u32 {
-        match h.get_parent(n) {
+        match h.base_hugr().get_parent(n) {
```

Comment:
This is a bit ugly (doing this on every call - alternative would be to do at every callsite, which is probably a bit better), but it kinda makes sense. (Better might be to calculate depth to the root of the *starting* HugrView, or something, but no way to do that)